packages/web-react/src/types/dictionaries.ts

```diff
@@ -0,0 +1,2 @@
+export type DictionarySizeType = 'small' | 'medium' | 'large';
```

Comment:
Move this file to `shared/dictionaries.ts`, please.

Reply:
ok